### PR TITLE
Added Headline and Button

### PR DESCRIPTION
**Pull request by BorchBe99**

### CODE REVIEW

**Comment by LennartRukower** [comment]
Please consider running tslint or use a tslint IDE extension before submitting to prevent style guide violations like:

- single quotes

- non arrow functions

- trailing whitespace

- missing semicolons

- unused imports

- etc.

**Comment by LennartRukower** [comment]
You should also link the corresponding issue(s).
_https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue_

File: client/src/ts/global/NotFound.tsx [hunk 1]
```diff
@@ -1,10 +1,23 @@
...
+import Button from '@material-ui/core/Button';
 
 const NotFound: React.FunctionComponent = () => {
-
+  /*
+          display: "flex",
+          justifyContent: "center",
+          alignItems: "center"
+  
+  */
```
Comment (LennartRukower) [inline client/src/ts/global/NotFound.tsx:10]: Please remove unnecessary code (there should be no commented out code).

File: client/src/ts/global/NotFound.tsx [hunk 2]
```diff
@@ -1,10 +1,23 @@
...
   return (
     <div className="App">
-      <h1>Error: Page not found!</h1>
+      <span style={{
```
Comment (LennartRukower) [inline client/src/ts/global/NotFound.tsx:13]: Don't use inline styling. 
Use the material-ui hook-api: _https://material-ui.com/styles/basics/#hook-api_

**Comment by LennartRukower** [comment]
The NotFound-page is not mobile responsive! 

**Always develop with mobile in mind (mobile first).**

**Comment by LennartRukower** [comment]
Also: 

- Always request a review from at least one of the other contributors, when creating a pr

- Please add a little description to your pr

File: client/src/ts/global/NotFound.tsx (hunk 1 above)
Comment (christophtrybek) [reply client/src/ts/global/NotFound.tsx:10]: Correct

File: client/src/ts/global/NotFound.tsx (hunk 2 above)
Comment (christophtrybek) [reply client/src/ts/global/NotFound.tsx:13]: Correct

**Comment by BorchBe99** [comment]
- Added mobile friendly scaling
- Fixed inline styling
- Fixed code comments

File: client/src/ts/global/NotFound.tsx [hunk 3]
```diff
@@ -1,10 +1,89 @@
...
+          <Typography variant="h5" component="h2">
+            Diese Seite wurde nicht gefunden.
+        </Typography>
+          <Typography className={classes.pos} color="textSecondary">
+            404
+        </Typography>
+          <Typography variant="body2" component="p">
+            Bei Problemen bitte melden bei: <a href="mailto:it@studentische-beratung.de">it@studentische-beratung.de</a>
+            <br />
+            {""}
```
Comment (christophtrybek) [inline client/src/ts/global/NotFound.tsx:80]: Remove empty string

File: client/src/ts/global/NotFound.tsx [hunk 4]
```diff
@@ -1,10 +1,89 @@
...
+            Fehler:
+        </Typography>
+          <Typography variant="h5" component="h2">
+            Diese Seite wurde nicht gefunden.
+        </Typography>
+          <Typography className={classes.pos} color="textSecondary">
+            404
+        </Typography>
+          <Typography variant="body2" component="p">
+            Bei Problemen bitte melden bei: <a href="mailto:it@studentische-beratung.de">it@studentische-beratung.de</a>
```
Comment (christophtrybek) [inline client/src/ts/global/NotFound.tsx:78]: Line length < 100

File: client/src/ts/global/NotFound.tsx [hunk 5]
```diff
@@ -1,10 +1,89 @@
...
+            <br />
+            {""}
+          </Typography>
+        </CardContent>
+        <CardActions>
+          <Button className={classes.backButton} size="medium" variant="contained" disableElevation href="./">Zurück zum Dashboard</Button>
```
Comment (christophtrybek) [inline client/src/ts/global/NotFound.tsx:84]: Line length < 100

File: client/src/ts/global/NotFound.tsx [hunk 6]
```diff
@@ -1,10 +1,89 @@
...
+    <div className={"content-page ${classes.root}"}>
+      <Card className={classes.notFoundCard} variant="outlined">
+        <CardContent>
+          <Typography className={classes.title} color="textSecondary" gutterBottom>
+            Fehler:
+        </Typography>
+          <Typography variant="h5" component="h2">
+            Diese Seite wurde nicht gefunden.
+        </Typography>
+          <Typography className={classes.pos} color="textSecondary">
```
Comment (christophtrybek) [inline client/src/ts/global/NotFound.tsx:74]: Combine this section with the one above into "404: Diese Seite wurde nicht gefunden."

File: client/src/ts/global/NotFound.tsx [hunk 7]
```diff
@@ -1,10 +1,89 @@
...
+  }));
 
+const NotFound: React.FunctionComponent = () => {
+  const classes = useStyles();
   return (
-    <div className="App">
-      <h1>Error: Page not found!</h1>
+    <div className={"content-page ${classes.root}"}>
+      <Card className={classes.notFoundCard} variant="outlined">
+        <CardContent>
```
Comment (christophtrybek) [inline client/src/ts/global/NotFound.tsx:67]: Please check the correctness of the indentation within this tags

File: client/src/ts/global/NotFound.tsx [hunk 8]
```diff
@@ -1,10 +1,89 @@
 import React from "react";
```
Comment (christophtrybek) [inline client/src/ts/global/NotFound.tsx:1]: Please Add a descriptive Comment above

**Review by christophtrybek — CHANGES_REQUESTED** [review]
Overall good job some small details need to addressed before merging

File: client/src/ts/global/NotFound.tsx (hunk 7 above)
Comment (BorchBe99) [reply client/src/ts/global/NotFound.tsx:67]: Could you tell me which specific tag do you mean?

Comment (christophtrybek) [reply client/src/ts/global/NotFound.tsx:67]: Sorry i meant the <CardContent> Tag but please make sure that the whole file is correctly indented

Comment (BorchBe99) [reply client/src/ts/global/NotFound.tsx:67]: I've checked it and pushed it.

**Comment by BorchBe99** [comment]
My last push should include all requested changes

**Review by christophtrybek — APPROVED** [review]
Good Job 🚀👍🏻

Please consider adding a meaningful Description of the Pull Request in the future.

**Review by LennartRukower — APPROVED** [review]
Good job

**Pull request merged**